DatadogCrashReporting/Sources/CrashReporting.swift

```diff
@@ -38,6 +38,10 @@ public final class CrashReporting {
 
             try core.register(feature: reporter)
 
+            if let plcr = PLCrashReporterPlugin.thirdPartyCrashReporter {
+                try core.register(backtraceReporter: BacktraceReporter(reporter: plcr))
+            }
```

Comment:
This might be more elegant, but would require revamping the startup sequence for `DatadogCrashReporting`. It still depends on [`CrashReportingPlugin`](https://github.com/DataDog/dd-sdk-ios/blob/6eab2d82441c4997ce05f9ecee7d64cc38517fc1/DatadogCrashReporting/Sources/CrashReportingPlugin.swift#L176), which was V1 dependency-inverter, no longer required in V2. This refactor is medium-size, hence I didn't want to do it for this PR.